packages/shared/src/common/array.ts

```diff
@@ -0,0 +1,4 @@
+export function wrapInArray(element: unknown): unknown[] {
```

Comment:
could do something like this instead?

```export function toArray<T>(input: T | T[]): T[] {
  return Array.isArray(input) ? input : [input];
}

Reply:
Yup! Saw it in one of your PRs and "borrowed" the function from there :P